src/test/suite/constants.ts

```diff
@@ -20,54 +22,89 @@ environment:
   sdk: '>=2.3.0 <3.0.0'     
 `;
 
-export const pubspecWithOverReactFormat = `
+export const orfDevDependency = `
```

Comment:
There's some renaming's here because the more I added, the more readers could assume what was going on so names could be condensed.